test/acceptance/integration/sth/aggregated.feature

```diff
@@ -32,12 +32,14 @@ Feature: get aggregated values with differents requests from sth
 
   @happy_path
   Scenario: get aggregates values using sth
-    Given service "test_happy_path", service path "/test", entity type "room", entity_id "room2", with attribute number "1", attribute name "random" and attribute type "celcius"
+    Given service "test_happy_path", service path "/test", entity type "room", entity_id "room2", with attribute number "1", attribute name "tempe" and attribute type "celcius"
     And receives a notification with attributes value "random number=2", metadata value "True" and content "json"
-    When ask for aggregates with method "sum" and resolution "day"
-      | date_type | value               |
-      | dateFrom  | 2015-02-14T00:00:00 |
-      | dateTo    | 2015-12-31T00:00:00 |
+    When ask for aggregates with queries parameters
+      | parameter  | value               |
+      | aggrMethod | sum                 |
+      | aggrPeriod | day                 |
+      | dateFrom   | 2015-02-14T00:00:00 |
+      | dateTo     | 2015-12-31T00:00:00 |
     Then I receive an "OK" http code
```

Comment:
LittleComment - Try to start always the steps in the same way, not sometimes in lowercase and other in uppercase